### DISCUSSION TITLE
SSC overridden my client data and idk how

### DISCUSSION THREAD

**Discussion by ClownyTheClown** [Pryaxis/TShock#2104] [Q&A] [answered]
I just logged on in my server and left, then to realize that my client character suddenly has all my SSC information.

My client character was at 0 playtime, 100 hp, and 10 mp. After exiting my server my client character has 1 day of playtime, 500 hp, and 200 mp, and has all my SSC armor, and items...

I'm not sure how this happened or what I've done. My character's is in the 'owner' group with its default permissions.

Now when I try to do it again, just by entering and leaving, the SSC no longer override my client character. Please help
I don't particular mind that my client character data is overwritten but I'd like to know how to replicate this so I can save my friends' SSC data to their characters.

**Comment by bartico6** [comment, marked as answer]
The playtime being changed is unnatural, but the health, mana and inventory being overridden is most likely the result of messing with SSC mode while ingame.

I'm not sure you should overwrite someone's singleplayer character with SSC data as that's easy to abuse and most likely will get SSC, as a Terraria functionality, in trouble, or removed, if reported to be abused.
